app/views/content/subjects/maths/_article.html.erb

```diff
@@ -127,7 +127,7 @@
     </p>
   </section>
 
-  <%= render SubjectSpecific::TextBlockWithOneImageComponent.new(
+  <%= render Campaign::TextBlockWithOneImageComponent.new(
     heading: "Getting into teacher training",
     colour: "yellow",
     image_path: "media/images/content/hero-images/0009.jpg",
```

Comment:
This image is quite small on the `/subjects/maths` page compared with current prod, both on mobile and desktop

It's not a huge showstopper - but the image definitely loses a bit of impact

![image](https://user-images.githubusercontent.com/87642394/196921533-6096cc4d-853c-4553-b3cc-a663a39fd7c6.png)